presidio-analyzer/tests/test_recognizer_registry.py

```diff
@@ -139,3 +141,50 @@ def test_when_remove_pattern_recognizer_then_item_removed():
 
     # Expects zero custom recognizers
     assert len(recognizer_registry.recognizers) == 0
+
```

Comment:
Consider adding a fail test, to detect when a malformed recognizer is attempted to be added, and an error should be returned

Reply:
Added